app/controllers/api/v1/event_procedures_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
         authorized_scope = policy_scope(EventProcedure)
         event_procedures = EventProcedures::List.result(
           scope: authorized_scope,
-          params: params.permit(:page, :per_page, :month, :year, :payd).to_h
+          params: params.permit(:page, :per_page, :month, :year, :payd, :hospital_name).to_h
```

Comment:
[Optional] - What do you think about something like:
```suggestion
          params: params.permit(:page, :per_page, :month, :year, :payd, hospital: { :name }).to_h
```

With something like this, we'll be more flexible to accept more data/filters about hospitals.